integrations/Braze/browser.js

```diff
@@ -172,28 +147,121 @@ class Braze {
       "gender",
       "lastname",
       "phone",
-      "facebook",
-      "twitter",
-      "first_name",
-      "last_name",
       "dob",
       "external_id",
       "country",
       "home_city",
       "bio",
-      "gender",
-      "phone",
       "email_subscribe",
       "push_subscribe",
     ];
 
+    // deep clone the traits object
+    const traits = cloneDeep(rudderElement.message.context.traits);
+
     reserved.forEach((element) => {
       delete traits[element];
     });
+    if (this.supportDedup && this.previousPayload !== null) {
+      const prevUserId = get(this.previousPayload, "rudderElement.message.userId");
```

Comment:
Does updating the userId, will create a new user? If true then, if the userId is updated apart from going with the new flow can't we go with the existing flow?

Reply:
yes, it will change, updated